### ISSUE TITLE
Delivering training data for ML

### ISSUE THREAD

**Issue by dhobern** [gbif/doc-camera-trap-guide#31]
This may be out of scope, but another use for the images may be in training classifiers. If a dataset associates images (or better still image segments) with human-validated species identifications, it may be perfect for developing ML models for classifying future images.

**Comment by peterdesmet** [comment]
Agree, and this use case is supported by Camtrap DP. Will see how we can mention this in the guide.

**Comment by ben-norton** [comment]
I think there is a distinction between releasing training datasets for computer vision and publishing camera trap datasets, where the former is out of scope. A training dataset can be derived from camera trap data, but it's an entirely separate set of files. For example, image files are a requirement for traing CV models. They are generally not required to use camera trap data for ecological modeling. Conflating the two could lead to confusion and poor model training. So I think it's best to keep the two issues separate.

**Comment by peterdesmet** [comment]
@ben-norton Camtrap DP is designed to also support the training for computer vision though (it is one of the driving use cases for @kbubnicki). But I agree that this being a GBIF guide, the focus is publishing for biological research. I do think we can mention that publishing camera trap datasets (especially as Camtrap DP) can _also_ support ML.

**Comment by ben-norton** [comment]
@peterdesmet I agree.

**Comment by peterdesmet** [comment]
ML is now mentioned as a use case in https://docs.gbif-uat.org/camera-trap-guide/en/#section-4

**Comment by ben-norton** [comment]
@peterdesmet I recommend referencing the coco format and the following training datasets  repo as an example:
https://lila.science/datasets

Coco
https://cocodataset.org/#home

**Comment by peterdesmet** [comment]
Thanks Ben, rather than in that section, I have now referenced http://lila.science in the glossary item for [machine learning](https://docs.gbif-uat.org/camera-trap-guide/en/#machine-learning). I did not reference the coco format, as I think it would take us to far and one can also share in Camtrap DP, which is COCO compatible.

**Comment by kbubnicki** [comment]
Hey @ben-norton!

As @peterdesmet mentioned above, Camtrap DP is COCO-compatible, at least in terms of how it stores the coordinates of annotated objects (bounding boxes). Additionally, with Camtrap DP, you can include images directly in your data package. However, more interesting and pragmatic option is to include accessible URLs (public or including private access-tokens) to media files in your data package. This way, you can store your media files, for example, in a cloud storage and download only those files that you need for AI model training. Moreover, Camtrap DP can provide you with more camera trap domain-specific metadata, such as all the details about the deployment of your camera traps or more detailed object annotations, including the sex and age of animals. You can use this metadata for your AI model training as well.

So, in summary, I do not see any advantage of the COCO format over Camtrap DP for training AI models with camera trap data. Perhaps the only advantage is that ML professionals are more familiar with COCO.
